captainhook_enabled.json

```diff
@@ -0,0 +1,62 @@
+{
+    "commit-msg": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\Regex",
+                "options": {
+                    "regex": "/^[A-Z]+-\\d+:?\\s/",
+                    "error": "Your commit message MUST start with a ticket ID"
+                }
+            }
+        ]
+    },
+    "pre-push": {
+        "enabled": false,
+        "actions": []
+    },
+    "pre-commit": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\AppBundle\\GitHooks\\Action\\CheckForBlacklistedWords",
+                "options": {
+                    "extensions": ["php", "twig"],
+                    "keyword_blacklist": ["var_dump", "dump"],
```

Comment:
We can add `dd` here too.

https://github.com/symfony/symfony/pull/26970

Reply:
Added in 5e788b5e